airflow-core/src/airflow/ui/src/layouts/Nav/UserSettingsButton.tsx

```diff
@@ -88,9 +90,19 @@ export const UserSettingsButton = ({ externalViews }: { readonly externalViews:
     <>
       <Menu.Root positioning={{ placement: "right" }}>
         <Menu.Trigger asChild>
-          <NavButton icon={FiUser} title={translate("user")} />
+          <NavButton icon={FiUser} title={translate("user.user")} />
         </Menu.Trigger>
         <Menu.Content>
+          {currentUser ? (
+            <Box borderBottom="1px solid" borderColor="border.muted" p={3}>
+              <Box color="fg.muted" fontSize="sm">
+                {translate("user.signedInAs")}
+              </Box>
+              <Box fontSize="md" fontWeight="semibold">
+                {currentUser.username}
+              </Box>
+            </Box>
+          ) : null}
```

Comment:
This could be undefined instead of null to fix this warning.

```
/home/karthikeyan/stuff/python/airflow/airflow-core/src/airflow/ui/src/layouts/Nav/UserSettingsButton.tsx
  105:15  warning  Use `undefined` instead of `null`  unicorn/no-null
```